doc/tutorials/lris_howto.rst

```diff
@@ -0,0 +1,485 @@
+
+    pypeit_chk_tilts Calibrations/Tilts_A_0_DET01.fits
+
+.. image:: ../figures/lris_tilts.png
+   :scale: 40%
+
+This shows a zoom-in of a :ref:`tiltimg` image in a `ginga`_ window with overlaid
+the 2D fitted tilts (blue), the masked pixels (red), and the pixels rejected in the 2D fitting (yellow).
+
+See :ref:`tilts` for further details.
+
+Flatfield
++++++++++
+
+PypeIt computes a number of multiplicative corrections to correct the 2D spectral response
+for pixel-to-pixel detector throughput variations and lower-order spatial and spectral illumination
+and throughput corrections.  We collectively refer to these as flat-field corrections; see
+:ref:`here <flat_fielding>` and :ref:`here <flat>`.
+To inspect the ``Flat`` images we can use the script :ref:`pypeit_chk_flats`, with this explicit call:
+
+.. code-block:: bash
+
+    pypeit_chk_flats Calibrations/Flat_A_0_DET01.fits
+
+Here is a zoom-in screenshot from the first tab in the `ginga`_ window (``pixflat_norm``):
+
+.. image:: ../figures/lris_flat.png
+   :scale: 40%
+
+This shows the normalized flat field image. The green/magenta lines are the slit edges, which are
+tweaked using the illumination flat field.
+
+
+See :ref:`flat` and :ref:`flat_fielding` for further details.
+
+Object finding
+--------------
+
+After the above calibrations are complete, PypeIt will iteratively identify
+sources, perform global and local sky subtraction, and perform 1D spectral
+extractions.  This process is fully described here: :ref:`object_finding`.
+
+PypeIt produces QA files that allow you to assess the detection of the objects.
+One example is ``QA\PNGs\r230417_01033-frb22022_LRISr_20230417T082242.672_DET01_S0212_obj_prof.png``:
+
+.. image:: ../figures/lris_objfind.png
+   :scale: 30%
+
+This shows the spatial profile of the object's S/N collapsed along the spectral direction.
+The dashed red line is the S/N threshold set by the :ref:`findobjpar`, and the green circle
+marks the spatial position of the detected object.
+
+Flexure
+-------
+
+PypeIt also measures a spectral flexure correction, by performing a cross-correlation
+between the sky spectrum extracted in each slit and an archived sky spectrum.
+The relative shift between the two spectra is then used to correct the wavelength solution
+for each slit (see :ref:`flexure-spectral`).
+Two flexure corrections are computed, one (called ``global``) using the sky lines
+extracted at the center of the slit, and one (called ``local``) using the sky lines
+extracted at the location of the science object.
+
+There are two QA files (two for the ``global`` and two for the ``local`` correction)
+that can be used to assess the flexure correction. Here is an example of the ``global`` correction,
+``QA/PNGs/r230417_01033-frb22022_LRISr_20230417T082242.672_global_DET01_S0212_spec_flex_sky.png``:
+
+.. image:: ../figures/lris_flexure_qa.png
+   :scale: 100%
+
+It shows sky spectrum cutouts around some of the sky lines used for the flexure correction.
```

Comment:
Same here.  Comment briefly on what one hopes/expects to see.

Reply:
done